test/controllers/organization_memberships_controller_test.exs

```diff
@@ -0,0 +1,179 @@
+defmodule CodeCorps.OrganizationMembershipControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.OrganizationMembership
+  alias CodeCorps.Organization
+  alias CodeCorps.User
+
+  import CodeCorps.Factories
+
+  def filter_params(records) do
+    ids =
+    records
+    |> Enum.map(fn(r) -> r.id end)
+    |> Enum.join(",")
+    %{filter: %{id: ids}}
+  end
+
+  setup do
+    conn =
+      %{build_conn | host: "api."}
+      |> put_req_header("accept", "application/vnd.api+json")
+      |> put_req_header("content-type", "application/vnd.api+json")
+
+    {:ok, conn: conn}
+  end
+
+  test "filters resources on index", %{conn: conn} do
+    membership_1 = insert(:organization_membership)
+    membership_2 = insert(:organization_membership)
+    insert(:organization_membership)
+
+    params = filter_params([membership_1, membership_2])
+    path = conn |> organization_membership_path(:index)
+
+    data = conn |> get(path, params) |> json_response(200) |> Map.get("data")
```

Comment:
Could the `path` and `data` be combined into pipes given that `conn` is the link between them and `data` is what we want at the end?